pkg/kubelet/kubelet.go

```diff
@@ -1900,6 +1901,88 @@ func (kl *Kubelet) cleanUpContainersInPod(podId types.UID, exitedContainerID str
 	}
 }
 
+// GetPodSandboxNetNS returns the network namespace path of the given podSandbox
```

Comment:
nit: can we move these functions to `pkg/kubelet/network.go` with comment explaining that these will be removed after switching to CRI?